app/views/pages/sicAndCompliance/financial/advice_or_consultancy.scala.html

```diff
@@ -0,0 +1,42 @@
+@import models.view.sicAndCompliance.financial.AdviceOrConsultancy
+@(adviceOrConsultancyForm: Form[AdviceOrConsultancy])(implicit request: Request[_], messages: Messages)
+@import uk.gov.hmrc.play.views.html.{helpers => govHelpers}
+@import views.html.helpers.errorSummary
+
+@main_template(title = messages("pages.financialCompliance.adviceOrConsultancy.title")) {
+
+    <div class="column-one-third">
+
+        <h1 class="form-title heading-xlarge" id="pageHeading">@messages("pages.financialCompliance.adviceOrConsultancy.heading")</h1>
+
+        @errorSummary(
+            Messages("app.common.errorSummaryLabel"),
+            adviceOrConsultancyForm
+        )
+
+        @govHelpers.form(action = controllers.sicAndCompliance.financial.routes.AdviceOrConsultancyController.submit()) {
+            <div class="form-group">
+                <fieldset class="inline">
+                    <span id="adviceOrConsultancyRadio"/>
+
+                    @govHelpers.inputRadioGroup(
+                        field = adviceOrConsultancyForm("adviceOrConsultancyRadio"),
+                        Seq(
+                            "true" -> Messages("app.common.yes"),
```

Comment:
this is a minor thing really, not worth fixing - the thing is this is making a static reference to Messages object rather than using the dependency-injected messages object. In the future it would be good if we lower-cased the Messages to "messages"